### DISCUSSION TITLE
Alarmo Integration using Envisalink_new entities

### DISCUSSION THREAD

**Discussion by jezzalencko** [ufodone/envisalink_new#117] [General]
envisalink_new has solved all my issues with previous envisalink, thank you!
I am running EVL4 MAX and on a Vista 48a panel managed through a Tuxedo Touch screen. All arm/disarm functionality is working with HA in-built alarm panel config.
Problem is, I'd like to leverage the Alarmo integration which is a lot more powerful. When integrating this however, it creates another entity **separate** from the one created by Envisalink. See attached.
<img width="1542" alt="Screenshot 2024-06-09 at 3 57 06 pm" src="https://github.com/ufodone/envisalink_new/assets/47110714/26dfa079-5ab7-4de1-8220-58427108a219">

Question is, however how can I run Alarmo but force it to use the entity created by Envisalink_new? Anyone with an idea how to do this, would be much appreciated. I'm sure it's something simple, but can't work it out.

**Comment by ufodone** [comment]
This is maybe a question better asked to the Alarmo integration folks.

If this was working for you with the stock `envisalink` integration, then the only thing I can think of would be that you might need to rename any config that references `envisalink` entities to `envisalink_new`.
